src/database/sql_database.h

```diff
@@ -236,6 +236,8 @@ class SQLDatabase : public Database {
             , operation(operation)
         {
         }
+        AddUpdateTable(const AddUpdateTable&) = delete;
+        AddUpdateTable(AddUpdateTable&&) = default;
```

Comment:
The C++ standard says if move operations are defined, copy operations get implicitly deleted.

I would also add an operator=(&&) here.

Reply:
https://en.cppreference.com/w/cpp/language/copy_constructor